File: SimpleCalculator/app/src/main/java/com/example/simplecalculator/MainActivity.java
package com.example.simplecalculator;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import android.widget.EditText;
import android.widget.TextView;

public class MainActivity extends AppCompatActivity {
    EditText n1;
    EditText n2;
    TextView result;


    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        n1 = (EditText) findViewById(R.id.Number_1);
        n2 = (EditText) findViewById(R.id.Number_2);
        result = (TextView) findViewById(R.id.Result_id);
        addFunction();
        subFunction();
        MultiplicationFunction();

        DivideFunction();
    }

    private void addFunction(){
        Button btn = findViewById(R.id.Addition);
        btn.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                try {
                    int num1 = Integer.parseInt(n1.getText().toString());
                    int num2 = Integer.parseInt(n2.getText().toString());
                    int sum = num1 + num2;
                    result.setText(Integer.toString(sum));

                }catch (Exception e){
                    result.setText("Enter Both Value");
                }

            }
        });

    }


    private void subFunction(){
        Button btn = findViewById(R.id.Subtraction);
        btn.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                try {
                    int num1 = Integer.parseInt(n1.getText().toString());
                    int num2 = Integer.parseInt(n2.getText().toString());
                    int sum = num1 - num2;
                    result.setText(Integer.toString(sum));
                }catch (Exception e){
                    result.setText("Enter Both Value");

                }
            }
        });

    }


    private void MultiplicationFunction(){
        Button btn = findViewById(R.id.multiplication);
        btn.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                try {
                    int num1 = Integer.parseInt(n1.getText().toString());
                    int num2 = Integer.parseInt(n2.getText().toString());
                    int sum = num1 * num2;
                    result.setText(Integer.toString(sum));
                }catch (Exception e){
                    result.setText("Enter Both Value");
                }
            }
        });

    }


    private void DivideFunction(){
        Button btn = findViewById(R.id.Divide);
        btn.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                try {
                    Double num1 = Double.parseDouble(n1.getText().toString());
                    Double num2 = Double.parseDouble(n2.getText().toString());
                    Double sum = num1 / num2;
                    result.setText(Double.toString(sum));
                }catch (Exception e){
                    result.setText("Enter Both Value");
                }
            }
        });

    }


}

File: SetCounter/app/src/main/java/com/example/setcounter/MainActivity.kt
package com.example.setcounter

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.widget.Button
import android.widget.TextView

class MainActivity : AppCompatActivity() {

    private lateinit var btn : Button
    private lateinit var textview : TextView
    private lateinit var btn2 : Button
    private lateinit var startBtn : Button
    private lateinit var pauseBtn : Button
    private lateinit var resetBtn : Button
    private var count : Int = 1
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        initializer()
        btnListener()    }
    private fun initializer(){
        btn = findViewById(R.id.button)
        textview =  findViewById(R.id.textView)
        btn2 = findViewById(R.id.button2)
        startBtn = findViewById(R.id.startBtn)
        pauseBtn = findViewById(R.id.pauseBtn)
        resetBtn = findViewById(R.id.resetBtn)
    }

    private fun btnListener(){
        btn2.setOnClickListener {
            count = 1
            textview.setText("Count")
        }
        btn.setOnClickListener {
            var c1 : Int =   count++
            textview.setText(c1.toString())

        }

    }
}